tap_freshdesk/streams.py

```diff
@@ -288,6 +292,13 @@ class Conversations(ChildStream):
     path = 'tickets/{}/conversations'
     parent = 'tickets'
 
+    def add_fields_at_1st_level(self, record):
+        """
+        Overwrite updated__at value.
+        """
+        if record.get("last_edited_at"):
+            record["updated_at"] = max(record["updated_at"], record["last_edited_at"])
```

Comment:
Please add comments here regarding why the above changes are needed, and we can't use updated_at directly.

Reply:
Added code comments to explain above `if` condition.